.gitmodules

```diff
@@ -50,3 +50,6 @@
 [submodule "third-party/small"]
 	path = third-party/small
 	url = https://github.com/transmission/small.git
+[submodule "third-party/rapidjson"]
+	path = third-party/rapidjson
+	url = https://github.com/Tencent/rapidjson.git
```

Comment:
#### Note to maintainers

I don't know when do we fork and when do we not fork. Just let me know if you've forked RapidJSON and wanted me to use that instead.

Reply:
>  Just let me know if you've forked RapidJSON and wanted me to use that instead.

@tearfur I've created https://github.com/transmission/rapidjson; feel free to use that.

Reply:
gtest seems to be an exception for some reason.

Anyway I'll be waiting for the fork if you agree with this change.

Reply:
wrt GTest, Looks like this was an oversight on my part @ 677dc73eac3efa769eefb1c7ec4eda232a35977a.

Reply:
I just followed your practice with small. It also has CMake support, but you chose to write our own `FindSmall.cmake` anyway.

Reply:
Those are two different topics, unless I'm misunderstanding. What I meant is I don't see the need to create a new stabilization branch in the rapidjson repo, since `master` seems to be pretty stable upstream

Reply:
Ah yes, then we're good here.